plugins/packageManager/controller.xql

```diff
@@ -1,4 +1,4 @@
-xquery version "1.0";
+xquery version "3.1";
```

Comment:
I believe XQuery 3.1 is incompatible with eXist 2.2. The package would have to be updated to declare the minimum requirement of eXist 3.0.